bin/lib/fetchContent.js

```diff
@@ -220,11 +223,17 @@ function articleAsItem(uuid) {
 }
 
 function articlesAsItems(uuids) {
-	const promises = uuids.map(articleAsItem);
-	return Promise.all( promises )
-	.then( outputs => {
-		return outputs.filter( op => {return op !== null;});
+	debug(`articlesAsItems: uuids.length=${uuids.length}, CAPI_CONCURRENCE=${CAPI_CONCURRENCE}`);
+
+	const articlePromisers = uuids.map( uuid => {
+		return function () {
+			return articleAsItem(uuid);
+		};
 	});
+
+	return directly(CAPI_CONCURRENCE, articlePromisers)
+	.then( outputs => outputs.filter( op => {return op !== null;}) )
```

Comment:
Up to you but you could use `.then( outputs => outputs.filter(op => op) )` for potential simples

Reply:
good to know, defo shorter, buuut a teensy bit more magic involved.